src/Data/DotLang/Attr/Edge.purs

```diff
@@ -27,7 +36,23 @@ instance attrDotLang :: DotLang Attr where
   toText (FontColor s) = "fontcolor=\"" <> toHexString s <> "\""
   toText (FontSize i) = "fontsize="<> show i
   toText (Style f) = "style="<> toText f
-  toText (Label t) = "label="<> show t
+  toText (Label (TextLabel t)) = "label=" <> show t
+  toText (Label (HtmlLabel t)) = "label=" <> t
   toText (FillColor c) = "fillcolor=\"" <> toHexString c <> "\""
   toText (PenWidth i) = "penwidth="<> show i
 
+-- |
+-- | ```purescript
+-- | htmlLabel "<table><tr><td>Label</td></tr></table>" -- :: Attr
+-- | ```
+-- | htmlLabel as a part of an attribute of an edge.
+htmlLabel :: String -> Attr
+htmlLabel = HtmlLabel >>> Label
+
+-- |
+-- | ```purescript
+-- | textLabel "..." -- :: Attr
+-- | ```
+-- | textLabel as a part of an attribute of an edge.
+textLabel :: String -> Attr
+textLabel = HtmlLabel >>> Label
```

Comment:
I'd prefer this to be just called `label` instead of `textLabel`
The reason is, that the users that previously used `Label` would no need to use `label`. If it's named the same, it's easier to migrate